.storybook/config.js

```diff
@@ -5,14 +5,14 @@ import {Provider as StyletronProvider} from 'styletron-react';
 import {Client as Styletron} from 'styletron-engine-atomic';
 
 import {ThemeProvider} from '../src/styles';
-import {LightTheme} from '../src/themes';
+import {LightThemeMove} from '../src/themes';
 
 // Add providers for theme and styletron
 const engine = new Styletron();
 addDecorator(story => {
   return (
     <StyletronProvider value={engine}>
-      <ThemeProvider theme={LightTheme}>{story()}</ThemeProvider>
+      <ThemeProvider theme={LightThemeMove}>{story()}</ThemeProvider>
```

Comment:
Do we want to replace it or add a switch to toggle between move and system fonts?

Reply:
we have a separate story to support theme switching for the docs site, I think

Reply:
this is also only for the storybook, which is only here for screener now